src/commands/output.rs

```diff
@@ -25,6 +27,65 @@ pub struct OutputFormatter {
 }
 
 impl OutputFormatter {
+    pub fn format_output(&self, response: MonitorResponse, status: TrackStatus, output_format: &str) -> Output {
+        let text_template = &self.config.text_template;
+        if let Some(track) = response.track {
+            let mut class = vec![];
+            let mut separator = "-";
+            if response.is_liked.unwrap_or_default() {
+                class.push("liked".to_string());
+                separator = "+";
+            }
+            class.push(status.into());
+            let text = match (track.artist, track.title) {
+                (Some(artist), Some(title)) => {
+                    let status_icon = match status {
+                        TrackStatus::Playing => &text_template.playing,
+                        TrackStatus::Paused => &text_template.paused,
+                        _ => "" // unable to get player state
+                    };
+                    let liked_icon = match class.contains(&"liked".to_string()) {
+                        true => &text_template.liked,
+                        _ => "" // not liked, so display no icon
```

Comment:
```suggestion
                    let liked_icon = if is_liked {
                    	&text_template.liked
                    } else {
                    	"" // Display no icon if not liked
                    }
```

The downside here is that you won't be able to control padding if it's liked or not. Though this is better than nothing for now. While I prefer  what I mentioned in my other comments, I think you're right in that it can be worked towards.

Reply:
The padding can be controlled via a combination of `format` and `text_template` settings, although it's a bit unintuitive. Here's an example:

```toml
format = "{status}{liked}{title} {separator} {artist}"
[text_template]
playing = "|> "
paused = "|| "
liked = "+ "
```